config/database.yml

```diff
@@ -18,7 +18,7 @@ development:
 # Do not set this db to the same as development or production.
 test:
   <<: *local_default
-  database: vita-min_test<%= ENV['TEST_ENV_NUMBER'] == '1' ? '' : ENV['TEST_ENV_NUMBER'] %>
+  database: vita-min_test<%= ENV['TEST_ENV_NUMBER'] %>
```

Comment:
Why did this change?

Reply:
We can revert this - it just seems unnecessary based on the documentation [here](https://github.com/grosser/parallel_tests?tab=readme-ov-file#add-to-configdatabaseyml). Seems like TEST_ENV_NUMBER is already the empty string for the first process (and I confirmed this in my testing), so we still get a database named vita-min_test when we do `rake parallel:create`. Maybe this isn't how it worked previously? In any case, I think it's safe to get rid of the conditional.